### PR TITLE
STN-730: Slider Styles

### PR DESCRIPTION
**Pull request by heatherdesigns**
# READY FOR REVIEW

## Summary
The work in this PR adds unique styles for the indicator dots and toggles on gradient hero slides. Adjustments to spacing and the alignment of the gradient hero slider component were also addressed.

The arrow toggles on the Colorful theme use the primary color pairing, and on Traditional use global-black.
The bars that indicate which slide you are on use secondary on Colorful, and global-black on Traditional.

Slick slider arrow toggle and dots / bars styles have been placed into Sass mixins to allow for easy implementation on future hero slider work.

Designs: [Figma comps](https://www.figma.com/file/BieDQhj8BoLlHJrzpDaIXV/Slider-Container-for-Hero-Components?node-id=0%3A1)

This work has been:
- [x] Documented in the Color Pairing Spreadsheet
- [x] Browser Tested
- [x] Tested for accessibility
- [x] Tested for logged in / logged out states

## Need Review By (Date)
5/13/2021

## Urgency
Medium

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. If you have a flexible page filled with examples of gradient hero sliders, you may use that for testing. If not, create a flexible page and add a variety of gradient hero sliders. I recommend including examples of the following:
- [x] Gradient Hero Slider in the hero area at the top of a page
- [x] Gradient Hero Slider in the main content section
- [x] Gradient Hero Slider with slides that have their alignment set to the right
- [x] Gradient Hero Sliders with a combination of fields (title, copy, and button) present and with a variation of content
- [x] Gradient Hero Slider in a row
- [x] Gradient Hero Slider in a well in a row (Note that at small screen sizes, all heros are squished. I don't think this is a common use case.)
3. Confirm the arrow toggles and slide indicators (bars) match the gradient hero designs comp.
4. Confirm the arrow toggles and slide indicators do not interfere with caption and credits on images.
5. Confirm the arrow toggles are keyboard accessible. Note that when you are testing keyboard navigation, once you tab to an arrow toggle, on your keyboard use the arrow key to move through the slides. You will not be able to navigate through the slides with the keyboard using the slide indicator bars. Slick JS adds a role of `presentation` to the bars (dots) which overrides the link behavior. I also recommend testing the keyboard functionality in the logged out state to avoid the Drupal admin contextual links.

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.slick.scss [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+          }
+
+          &:hover {
+            transform: none;
+          }
+        }
+      }
+    }
+  }
+}
```
Comment (rkelbel) [inline docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.slick.scss:214]: Mixin additions were fantastic ideas for this and should make things easy to update going forward!

File: docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.slick.scss [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+          bottom: calc(50% - #{hb-calculate-rems(46px)});
+        }
+      }
+    }
+  }
+}
+
+// dots aka bars
+@mixin slick-dots {
+  .slick-dots {
```
Comment (rkelbel) [inline docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.slick.scss:140]: I LOVE these slick dots, the size and shape, the color and subtle animation when selecting. Really well done Heather!

**Comment by rkelbel** [comment]
This component has really elevated the Gradient hero look, which already looked really sleek.  Just fantastic work Heather I love it!

I had one question which I believe is actually just refinement work we will do in another card, but just to confirm is the height adjustment on slide due to content size something we will look at for possible refinement?  I would assume it is as that would be out of scope for this work right now.
http://g.recordit.co/0CsO1TiDie.gif

This work is really exciting to see take shape, well done in both design and execution of the styles Heather! 🎉

**Comment by heatherdesigns** [comment]
> This component has really elevated the Gradient hero look, which already looked really sleek. Just fantastic work Heather I love it!
> 
> I had one question which I believe is actually just refinement work we will do in another card, but just to confirm is the height adjustment on slide due to content size something we will look at for possible refinement? I would assume it is as that would be out of scope for this work right now.
> http://g.recordit.co/0CsO1TiDie.gif
> 
> This work is really exciting to see take shape, well done in both design and execution of the styles Heather! 🎉

Yes! ✅  The height adjustment will involve changing the default functionality of the slider via custom JavaScript which will be addressed in future work.

**Review by rkelbel — APPROVED** [review]
This looks amazing Heather!

high five

**Pull request merged**